README.md

```diff
@@ -44,7 +44,7 @@ For example, the following command will create a Docker cluster with Cilium CNI
 ```shell
 export CLUSTER_NAME=docker-cluster-cilium-helm-addon
 export CLUSTER_FILE=examples/capi-quick-start/docker-cluster-cilium-helm-addon.yaml
-export KUBERNETES_VERSION=v1.28.7
+export KUBERNETES_VERSION=v1.29.4
```

Comment:
Thanks for the reminder, last time I checked the DNS pod was not coming up in a CAPA cluster, I didn't get a chance to debug.

Reply:
Ah ok. I'll run the e2e and have a look

Reply:
Yep failure - but why???

```
$ k --kubeconfig workload-kubeconfig logs -n kube-system coredns-76f75df574-lvqsr
Listen: listen tcp :53: bind: permission denied
```

Reply:
I think this is the issue - https://github.com/kubernetes/minikube/pull/18421

Reply:
I wonder if we should patch containerd config with https://github.com/kubernetes-sigs/kind/pull/2748#issuecomment-1132425822 - this is the default in containerd 2.0 btw when it gets released...

Reply:
This fixed the issue. I'll open a PR for discussion.

Reply:
See #645